backend/README.md

```diff
@@ -2,19 +2,26 @@
 
 ## Backend Development Setup
 
+### Local Database
+
 In root directory, run
 `docker compose up backend-dev --build --attach-dependencies` to start a
```

Comment:
The new command is:
`docker compose up --profile backend-dev up --build --attach-dependencies`

Reply:
Thanks, updated.